CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 * Exit indications are now drawn accurately with a correct exit bearing. [#1288](https://github.com/mapbox/mapbox-navigation-ios/pull/1288)
 * Added a delegate method, `NavigationViewControllerDelegate.navigationViewController(_:roadNameAt:)` which allows you to customize the contents of the road name label displayed towards the bottom of the map view. [#1309](https://github.com/mapbox/mapbox-navigation-ios/pull/1309)
 
+### User interface
```

Comment:
There are other UI-related changes above. Before or after merging, let’s recategorize everything, since the “master” section has gotten a bit long.